products/dns-security/sidebars.js

```diff
@@ -1,19 +1,3 @@
 module.exports = {
-  dns_api: [
-    {
-      type: "category",
-      label: "DNS Security API",
-      link: {
-        type: "generated-index",
-        title: "DNS Security API",
-        slug: "/dns-security/api"
-      },
-      items: [
-        {
-          type: "autogenerated",
-          dirName: "dns-security/api"
-        }
-      ]
-    }
-  ]
-};
+  dns_api: ["dns-security/api/dnssecurityapioverview", require("./api/sidebar")],
```

Comment:
If you want to use the ./api/sidebar, you need to update the docusaurus.config.js configuration for the dns-security api to have sidebarOptions like below: 

```
"dns-security": {
            specPath: "openapi-specs/dns-security/dns-security.yaml",
            outputDir: "products/dns-security/api",
            sidebarOptions: { groupPathsBy: "tag", categoryLinkSource: "info" },
          },